android-project/app/build.gradle

```diff
@@ -35,36 +34,37 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-    applicationVariants.all { variant ->
-        tasks["merge${variant.name.capitalize()}Assets"]
-            .dependsOn("externalNativeBuild${variant.name.capitalize()}")
+
+    if (buildAsApplication) {
+        applicationVariants.all { variant ->
+            tasks["merge${variant.name.capitalize()}Assets"]
+                .dependsOn("externalNativeBuild${variant.name.capitalize()}")
+        }
     }
+
     if (!project.hasProperty('EXCLUDE_NATIVE_LIBS')) {
         sourceSets.main {
             jniLibs.srcDir 'libs'
         }
         externalNativeBuild {
             ndkBuild {
-                path 'jni/Android.mk'
+                path '../../Android.mk'
             }
             // cmake {
             //     path 'jni/CMakeLists.txt'
             // }
         }
-       
+
     }
     lintOptions {
         abortOnError false
     }
-    
+
     if (buildAsLibrary) {
         libraryVariants.all { variant ->
-            variant.outputs.each { output ->
-                def outputFile = output.outputFile
-                if (outputFile != null && outputFile.name.endsWith(".aar")) {
-                    def fileName = "org.libsdl.app.aar";
-                    output.outputFile = new File(outputFile.parent, fileName);
-                }
+            variant.outputs.all {
+                def fileName = "org.libsdl.app.aar";
+                outputFileName = fileName
```

Comment:
I don't know much about building an aar, but I guess it's based on this.
https://stackoverflow.com/a/26555036

So it just renames the library to whatever you specify in `fileName`. Looks okay to me.